src/main/java/gameframework/motion/GameMovable.java

```diff
@@ -4,15 +4,26 @@
 
 import java.awt.Point;
 
+/**
+ * 
+ * Class of a movable object in the Canvas
+ */
 public abstract class GameMovable implements ObjectWithBoundedBox {
 	protected GameMovableDriver moveDriver ;
 	protected Point position = new Point();
 	protected SpeedVector speedVector = SpeedVector.createNullVector();
-
+	
+	/**
+	 * Default Constructor, create a Default Implementation of GameMovable
```

Comment:
We can see it's the default constructor. Explain how the resulting object will behave.